Automattic/wp-calypso
# Implement a new Calypso Environment for Automattic for Agencies

Resolves https://github.com/Automattic/jetpack-genesis/issues/213

## Proposed Changes

- Add new environment files for Automatic For Agencies: `a8c-for-agencies-development`, `a8c-for-agencies-stage`& `a8c-for-agencies-production`
- Add a new section for `a8c-for-agencies` with a dummy page that renders `Hello, World!`.
- Add server pages so that we show environment badges.
- Add new scripts to start the server, docker build, etc.
- Add new URLS as allowed origins.

## Testing Instructions

- Apply this patch: D137537-code
- Switch to the branch `add/implement-a4a-calypso-environment`.
- Start the server by running `start-a8c-for-agencies`
- Use the live links and verify both Calypso Blue & Green loads fine.
- Visit http://agencies.localhost:3000 and verify that you can see the page as shown below:

<img width="1728" alt="Screenshot 2024-02-08 at 7 39 42 PM" src="https://github.com/Automattic/wp-calypso/assets/10586875/c22d21b2-7873-4318-8c95-f779f7089f2c">


## Pre-merge Checklist
